packages/shared_preferences/shared_preferences_linux/.metadata

```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled and should not be manually edited.
+
+version:
+  revision: e491544588e8d34fdf31d5f840b4649850ef167a
```

Comment:
This was copied from another plugin - should this file exist/what revision should it be set to?

Reply:
It should exist; `flutter create` would have created it. If you copied from another plugin, then this revision is correct. (It's the Flutter hash for the version of Flutter that created the plugin).